README.md

```diff
@@ -32,10 +31,16 @@ To run using docker you:
 - setup environment variables
 
 #### Running from CLI
-`$ locust -f ./scale_tests/upload_cloud.py --host=local:1234 --no-web --client=100 --hatch-rate=50 --run-time=10s --csv=./scale_tests/upload`
+
+Using Python 3.6:
+
+    $ pip install -r requirements.txt
+    $ locust -f ./scale_tests/upload_cloud.py --host=$HOST --no-web --client=100 --hatch-rate=50 --run-time=10s --csv=./scale_tests/upload
+
+Where `$HOST` is the base URL of the DSS endpoint you want to hit, e.g. `https://dss.dev.data.humancellatlas.org/v1/`. (The tests will look for `${HOST}swagger.yaml`.)
```

Comment:
@DailyDreaming is there a convention for keeping the leading slash with the file name? or having it at the host level as a trailing slash?

Reply:
For reference, I am referring to this line: https://github.com/HumanCellAtlas/data-store-scale-test/blob/2223be80cc1cfa515dfcbbe147bf7def31359844/locustfiles/common/dsslocust.py#L201